scss/_popover.scss

```diff
@@ -23,19 +23,22 @@
   &.bs-tether-element-attached-bottom {
     margin-top: -$popover-arrow-width;
 
-    .popover-arrow {
-      bottom: -$popover-arrow-outer-width;
+    &::before,
```

Comment:
@mdo prefer the single-colon form for some reason. Please adjust these accordingly.